apps/cowswap-frontend/src/cowSdk.ts

```diff
@@ -10,4 +12,15 @@ export const metadataApiSDK = new MetadataApi()
 export const orderBookApi = new OrderBookApi({
   env: isBarnBackendEnv ? 'staging' : 'prod',
   ...(prodBaseUrls ? { baseUrls: prodBaseUrls } : undefined),
+  backoffOpts: {
+    ...DEFAULT_BACKOFF_OPTIONS,
+    retry(e: any, attemptNumber: number): boolean | Promise<boolean> {
+      // Retry only once when browser is offline
+      if (e?.message === FAILED_FETCH_ERROR && attemptNumber > 1) {
```

Comment:
can we infer that is offline becasue of failed fetch message?

We also have a subscription to the online state changes, and in principle we stop doing requests.

Reply:
Yes, it's actually not relevant anymore, removed it. Thanks!